docs/en/latest/plugins/skywalking.md

```diff
@@ -78,20 +79,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-You also can complete the above operation through the web interface, first add a route, then add skywalking plugin:
+You also can complete the above operation through the web interface, first add a route, then add SkyWalking plugin:
```

Comment:
You also can => You can also

Reply:
This following line exists widely in many APISIX docs - You also can complete the above operation through the web interface, should I change it for this one? Note that this also can, though sounds a bit odd, it is indeed a correct usage identical to can also.

FYI @tokers